### PR TITLE
Upgrade various libraries.

### PR DESCRIPTION
**Pull request by xiaodong-izettle**
F.Y.I @iZettle/horizons...

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,11 +58,12 @@
 
         <assertj-core.version>3.4.1</assertj-core.version>
         <astyanax.version>3.9.0</astyanax.version>
-        <aws.version>1.10.77</aws.version>
-        <bouncycastle.version>1.54</bouncycastle.version>
+        <aws.version>1.11.60</aws.version>
```
Comment (alorlea) [inline pom.xml]: I like this change :+1:

**Comment by xiaodong-izettle** [comment]
Ping @iZettle/horizons, thanks...

File: izettle-cart/pom.xml [hunk 2]
```diff
@@ -15,8 +15,8 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```
Comment (adamvoncorswant) [inline izettle-cart/pom.xml]: I don't think this will work for our Android devs...

Comment (adamvoncorswant) [inline izettle-cart/pom.xml]: Well, I think they're almost compatible with 1.7, but not all features (such as try-with-resources for example). So, while bumping source version wouldn't break compatibility itself, an unknowing backend coder could all of a sudden add code that won't work on android.

Thanks

Comment (fredrikbackstrom) [inline izettle-cart/pom.xml]: Doesn't Android support J7 and sort of J8? https://developer.android.com/guide/platform/j8-jack.html

Comment (adamvoncorswant) [inline izettle-cart/pom.xml]: Sort of, but not completely, and especially not API version 18 and below (which we still support)

**Comment by xiaodong-izettle** [comment]
@adamvoncorswant, could you please merge this PR if it's OK? :-)

**Comment by xiaodong-izettle** [comment]
@alorlea, since you are still in the office I assume?

**Comment by alorlea** [comment]
Let me start the laptop

**Comment by xiaodong-izettle** [comment]
Ah, sorry for the hassle... :-)

**Comment by alorlea** [comment]
Merging

**Comment by xiaodong-izettle** [comment]
Thanks, @alorlea. :)

**Comment by adamvoncorswant** [comment]
Ah, thanks guys. Sorry for bad latency :)

**Pull request merged**